metron-deployment/amazon-ec2/conf/defaults.yml

```diff
@@ -15,6 +15,22 @@
 #  limitations under the License.
 #
 ---
+# which services should be started?
+services_to_start:
+  - mysql
+  - elasticsearch
+  - pcap-service
+  - metron-ui
+  - yaf
+  - snort
+  - snort-logs
+  - bro
+  - pcap-replay
+  - yaf-parser
+  - bro-parser
+  - snort-parser
+  - enrichment
```

Comment:
For a future PR, this might be a nice pattern to use to define which services to install.  This would be in lieu of the various booleans that we use currently; `install_bro`, `install_elasticsearch`, etc.  A list is more compact and intuitive.